app/src/main/java/com/hedvig/app/feature/chat/ui/AttachFileAdapter.kt

```diff
@@ -132,7 +132,7 @@ class AttachFileAdapter(
                 uploadFile: (Uri) -> Unit
             ) {
                 binding.apply {
-                    val image = attachImageData[position - 1]
+                    val image = attachImageData[absoluteAdapterPosition - 1]
```

Comment:
```suggestion
                    val image = attachImageData[bindingAdapterPosition - 1]
```

They are the same in this case, but for the list to line up if it's a sub-adapter, this is needed 😉